pom.xml

```diff
@@ -131,13 +131,13 @@
       <dependency>
         <groupId>io.projectreactor</groupId>
         <artifactId>reactor-test</artifactId>
-        <version>3.3.1.RELEASE</version>
+        <version>3.3.5.RELEASE</version>
```

Comment:
We might should import their BOM (bill of materials): https://projectreactor.io/docs